R/module_filter_data.R

```diff
@@ -92,3 +92,12 @@ srv_filter_data <- function(id, datasets, active_datanames, data_rv, is_active)
 
   trigger_data
 }
+
+#' @rdname module_filter_data
+.get_filter_expr <- function(datasets, datanames) {
```

Comment:
needed to handle empty `datanames` - when `teal_data` without objects is submitted